superdesk/tests/steps.py

```diff
@@ -989,7 +993,9 @@ def when_upload_patch_dictionary(context):
     assert_ok(context.response)
 
 
-def upload_file(context, dest, filename, file_field, extra_data=None, method='post', user_headers=[]):
+def upload_file(context, dest, filename, file_field, extra_data=None, method='post', user_headers=None):
+    if user_headers is None:
+        user_headers = []
```

Comment:
any idea if we can enable some flake8 check for that pattern to avoid it in future?

Reply:
I found it only for pylint. There is a plugin for flake8 which may help with it. I will check it and if it works I'll create a new pr.